evansmusomi/react-native-102
# Adds Logout Ability

*What this does*
- User can log out of the app

*How to verify*
- When a user presses the logout button on the side drawer, they should be redirected to the Auth screen and unable to auto sign in until they log in again
